setup.py

```diff
@@ -100,6 +100,7 @@ def extra(self):
     extras_require={
         'ujson': ['ujson>=1.35'],
         'email': ['email-validator>=1.0.3'],
+        'typing_extensions': ['typing_extensions']
```

Comment:
```suggestion
        'typing_extensions': ['typing_extensions>=3.7.2']
```
Just to avoid surprises, unless we know of an earlier version we need to support.